app/classes/Transvision/ShowResults.php

```diff
@@ -321,6 +325,11 @@ public static function resultsTable($search_object, $search_results, $page)
                 if we aren't in the 3locales view and if we have a $target_string
             */
             $transliterate = $locale2 == 'sr' && ! $extra_locale && $target_string && $target_string != '@@missing@@';
+            if ($current_repo === 'aurora' && $toolUsedByTargetLocale == 'locamotion') {
+                $edit_link = "https://mozilla.locamotion.org/{$locale2}/firefox/translate/{$fileAndRawString[0]}.po?search={$fileAndRawString[1]}&sfields=locations";
```

Comment:
This link is not working, you need a `#` before `search`.

Example: https://mozilla.locamotion.org/eu/firefox/translate/browser/chrome/browser/aboutPrivateBrowsing.dtd.po#search=aboutPrivateBrowsing.info.cookies&sfields=locations